<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>601</width>
     <height>401</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QLabel" name="label_status">
    <property name="font">
     <font>
      <family>Nirmala UI Semilight</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Not Started</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Share forlder directory</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="text">
     <string>Sharing dir:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="text">
     <string>./</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="text">
     <string>Connection Address:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="text">
     <string>0.0.0.0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
